src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs

```diff
@@ -87,11 +87,47 @@ protected static string GetOrMapDiagnosticId(Diagnostic diagnostic, out bool inc
             return diagnostic.Id;
         }
 
+        protected abstract SyntaxNode GetContainingStatement(SyntaxToken token);
+        protected abstract bool TokenHasTrailingLineContinuationChar(SyntaxToken token);
+
         protected virtual SyntaxToken GetAdjustedTokenForPragmaDisable(SyntaxToken token, SyntaxNode root, TextLineCollection lines, int indexOfLine)
-            => token;
+        {
```

Comment:
Logic for this method and the below was moved from VB.NET impl, and translated

Reply:
can you doc what it's doing?